rust/src/python/libnmstate/prettystate.py

```diff
@@ -42,6 +40,18 @@
     Interface.KEY,
 )
 
+try:
+    from warnings import deprecated
+except ImportError:
+
+    class deprecated:
+        def __init__(self, message=None):
+            self.message = message or "This class is deprecated."
+
+        def __call__(self, cls):
+            print(self.message)
```

Comment:
Instead of printing, could we raise the warning? Like
```
        warnings.warn(
            "PrettyState class is deprecated;to be removed in future release."
            "The output of libnmstate.show() is already sorted.",
            DeprecationWarning,
        )
```